src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/CommonConditions.java

```diff
@@ -11,6 +13,7 @@
 import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.RESP_SOLICITOR_NAME;
 import static uk.gov.hmcts.reform.finrem.caseorchestration.service.CommonFunction.nullToEmpty;
 
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
Part of me thinks it would be better to just have a no-args constructor here instead of Lombok.
Happy to leave this as it is, though.

Reply:
I had that discussion with @jakub-kozlowski and @jacmurphy in other PR.

The things are:
1. code smell in sonar (as far as I know we can't change the config of sonar) - not really important, but metrics
2. code coverage - not really important, but metrics
3. you can't instantiate this class - which is what we want to achieve (not that we would like to, but just to make it clear what is the way of using it).

Reply:
I get that. All I'm saying is that this could also be achieved by something like this:
`private CommonConditions(){}`

Reply:
Well, yeah. In that case I think this looks better:

```@AllArgsConstructor(access = AccessLevel.PRIVATE)```

but it's just a personal preference